tooltester/src/main/resources/cwltoolPlaybook.yml

```diff
@@ -5,7 +5,7 @@
     - name: Install JDK
       become: yes
       apt:
-        name: default-jre
+        name: openjdk-8-jdk
```

Comment:
That's interesting. Have you been testing with openjdk 8 as opposed to oracle 8?

Reply:
yes

Reply:
yes